### ISSUE TITLE
Feature Request - Increase record limit when logged in?

### ISSUE THREAD

**Issue by mvzhuang** [ArctosDB/arctos#6650]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
I'd like to download more than 500 records without having to do special requests
**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_
Trying to download about 1700 records for a researcher.

**Describe the solution you'd like**
Increase record limit in logged in search to at least a few thousand. 
**Describe alternatives you've considered**
Use the request data 
**Additional context**
Is there a reason the public record search is 250,000 but logged in, I only have 500? I feel like we should be able to download at least 2000 with relatively few issues. 
**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by mvzhuang** [comment]
Ohhhh ok. Thanks Dusty! I didn't know about the cost thing. That is nifty to help prevent crashes.
